dashboard/app/controllers/featured_projects_controller.rb

```diff
@@ -41,23 +41,20 @@ def destroy
 
   # Featured projects are selected internally for their
   # quality, so we can be reasonably confident that they
-  # are not abusive. To prevent users from spamming Zendesk
-  # with false reports of abuse on featured projects, this
-  # sets their abuse score such that the project needs to
-  # be reported many times before being blocked.
-  def buffer_abuse_score(score = -50)
+  # are not abusive. We reset their abuse score to 0.
+  def reset_abuse_score(score = 0)
```

Comment:
should we also get rid of the parameter here?

Reply:
Updated!